include/pressio/ops/tpetra/ops_level2.hpp

```diff
@@ -96,6 +97,7 @@ _product_tpetra_mv_sharedmem_vec(const alpha_t & alpha,
 template <class A_type, class x_type, class y_type, class alpha_t, class beta_t>
 ::pressio::mpl::enable_if_t<
   ::pressio::is_multi_vector_tpetra<A_type>::value and
+  ::pressio::is_vector_kokkos<x_type>::value and
   ::pressio::is_vector_tpetra<y_type>::value
   >
 _product_tpetra_mv_sharedmem_vec_kokkos(const alpha_t & alpha,
```

Comment:
please remove `_kokkos`

Reply:
thanks for catching this, fixed on 7247774710e3c0b08d6081210e957d81a602b3f3